pkg/web/templates/components/transaction_rejection.html

```diff
@@ -31,16 +32,27 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
         </div>
         <div class="my-4">
           <label for="notes" class="label-style">Notes</label>
-          <textarea id="notes" name="notes" rows="4" class="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500"></textarea>
+          <textarea id="notes" name="notes" rows="4"
```

Comment:
Update to match API:

```suggestion
          <textarea id="notes" name="message" rows="4"
```